cmd/opentelemetry-collector/app/exporter/kafkaexporter/doc.go

```diff
@@ -12,5 +12,5 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package kafkaexporter implements Jaeger Kafka exporter.
+// Package kafkaexporter implements Jaeger Kafka ingester as OpenTelemetry exporter.
```

Comment:
This should be `producer` rather than `ingester`, to be consistent with the kafka receiver.